Release notes YDB Server 24.2.7 (#12729)
ydb-platform/ydb: ydb/docs/en/core/changelog-server.md

* Enabled autocomplete in the queries editor by default.
* Added support for views.

### Bug fixes

* Added a check on the size of the local transaction before its commit to fix [errors](https://github.com/db-platform/ydb/issues/6677) in the work of schematic operations when export/backup large databases.
* [Fixed](https://github.com/ydb-platform/ydb/pull/7709) duplicate results in select queries when reducing quotas in [DataShard](./concepts/glossary#data-shard).
* [Fixed](https://github.com/ydb-platform/ydb/pull/6461) [errors](https://github.com/ydb-platform/ydb/issues/6220) occurring during [coordinator](./concepts/glossary#coordinator) state changes.
* [Fixed](https://github.com/ydb-platform/ydb/pull/5992) issues during the initial CDC scan.
* [Resolved](https://github.com/ydb-platform/ydb/pull/6615) race conditions in asynchronous change delivery (asynchronous indexes, CDC).
* [Fixed](https://github.com/ydb-platform/ydb/pull/5993) a crash that sometimes occurred during [TTL-based](./concepts/ttl) deletions.
* [Fixed](https://github.com/ydb-platform/ydb/pull/5760) an issue with PDisk status display in the [CMS](./concepts/glossary#cms).

> * [Fixed](https://github.com/ydb-platform/ydb/pull/6008) a bug where soft tablet transfer (drain) from a node could hang.

Reviewer: ```suggestion
* [Fixed](https://github.com/ydb-platform/ydb/pull/6008) an issue that might cause soft tablet transfers (drain) from a node to hang.
```

> * [Resolved](https://github.com/ydb-platform/ydb/pull/6445) a problem where the interconnect proxy on a node without restarts would stop when adding another node to the cluster.

Reviewer: ```suggestion
* [Resolved](https://github.com/ydb-platform/ydb/pull/6445) an issue with the interconnect proxy stopping on a node that is running without restarts. The issue occurred when adding another node to the cluster.
```

* [Corrected](https://github.com/ydb-platform/ydb/pull/7023) string escaping in error messages.

> * [Fixed](https://github.com/ydb-platform/ydb/pull/6695) free memory accounting in the [interconnect](./concepts/glossary#actor-system-interconnect).

Reviewer: ```suggestion
* [Fixed](https://github.com/ydb-platform/ydb/pull/6695) an issue with managing free memory in the [interconnect](./concepts/glossary#actor-system-interconnect).
```

* [Corrected](https://github.com/ydb-platform/ydb/issues/6405) UnreplicatedPhantoms and UnreplicatedNonPhantoms counters in VDisk.

> * [Fixed](https://github.com/ydb-platform/ydb/issues/6398) handling of empty garbage collection requests on VDisk.

Reviewer: ```suggestion
* [Fixed](https://github.com/ydb-platform/ydb/issues/6398) an issue with handling empty garbage collection requests on VDisk.
```

* [Resolved](https://github.com/ydb-platform/ydb/pull/5894) issues with managing TVDiskControls settings through CMS.

> * [Fixed](https://github.com/ydb-platform/ydb/pull/5883) a bug where data created by newer versions of VDisk failed to load.

Reviewer: ```suggestion
* [Fixed](https://github.com/ydb-platform/ydb/pull/5883) an issue with failing to load the data created by newer versions of VDisk.
```

> * [Fixed](https://github.com/ydb-platform/ydb/pull/5862) execution of `REPLACE INTO` queries with default values.

Reviewer: ```suggestion
* [Fixed](https://github.com/ydb-platform/ydb/pull/5862) an issue with executing the `REPLACE INTO` queries with default values.
```
